### PR TITLE
💄Fix broken layout due to long folder/file name (#117 & #124)

### PR DESCRIPTION
**Pull request by Aymerix01**
## Description

Two issues were fixed:
The first one was when folder/file name were really long, the right layout with the 3 dots button was not on the screen. Now, if the name is too long, this button is still visible but not the whole name (in fact the name goes under the button and doesn't push it away)
Folder,file with long name

The second one was when a folder had also a long name and you entered into it, the access path could also push the right layout out of the screen. Now, if the folder name is too long, it ends with ... in the access path to reduce it.
access path

### CODE REVIEW

File: tdrive/frontend/src/app/views/client/body/drive/documents/document-row.tsx [hunk 1]
```diff
@@ -104,10 +104,10 @@ export const DocumentRow = ({
           }
         />
       </div>
-      <div className="grow text-ellipsis whitespace-nowrap overflow-hidden">
+      <div className="grow text-ellipsis whitespace-nowrap overflow-hidden" style={{ width: '100px'}}>
```
Comment (shepilov) [inline tdrive/frontend/src/app/views/client/body/drive/documents/document-row.tsx]: do not use absolute values at all please, absolute values are eligible only for min and max style,

File: tdrive/frontend/src/app/views/client/body/drive/documents/document-row.tsx [hunk 2]
```diff
@@ -104,10 +104,10 @@ export const DocumentRow = ({
...
         <Base>{item.name}</Base>
       </div>
-      <div className="shrink-0 ml-4">
+      <div className="shrink-0 ml-4" style={{ width: '20px' }}>
```
Comment (shepilov) [inline tdrive/frontend/src/app/views/client/body/drive/documents/document-row.tsx]: and also, could you move the styles, to the styles file, or what is even better if it's possible to use existing classes from tailwind css

File: tdrive/frontend/src/app/views/client/body/drive/header-path.tsx [hunk 3]
```diff
@@ -25,6 +25,18 @@ export default ({
...
 };
 
+function checkNamelength (name: any){
+  if (typeof name !== "undefined" ){
+    if (name.length >= 30){
+      return name.substring(0,30)+" ...";
+    } else {
+      return name;
+    }
+  }else {
```
Comment (chibenwa) [inline tdrive/frontend/src/app/views/client/body/drive/header-path.tsx]:
```suggestion
  } else {
```

File: tdrive/frontend/src/app/views/client/body/drive/header-path.tsx [hunk 4]
```diff
@@ -25,6 +25,18 @@ export default ({
   return <PathRender inTrash={inTrash || false} path={path} onClick={id => setParentId(id)} />;
 };
 
+function checkNamelength (name: any){
```
Comment (chibenwa) [inline tdrive/frontend/src/app/views/client/body/drive/header-path.tsx]:
```
function checkNamelength (name: any){
```

This is a bad name.

What are you "checking" for? Is checking even the best name here?

**Review by shepilov — APPROVED** [review]

**Pull request merged**